src/test/suite/didact.test.ts

```diff
@@ -21,7 +21,8 @@ import { before, beforeEach, after } from 'mocha';
 import * as fs from 'fs';
 import * as vscode from 'vscode';
 import * as path from 'path';
-import {SCAFFOLD_PROJECT_COMMAND, START_DIDACT_COMMAND, extensionFunctions} from '../../extensionFunctions';
+import * as extensionFunctions from '../../extensionFunctions';
+import {SCAFFOLD_PROJECT_COMMAND, START_DIDACT_COMMAND} from '../../extensionFunctions';
```

Comment:
why 2 different imports for the same class?

Reply:
good catch